### PR TITLE
Add community oVirt counter

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Changes proposed in this pull request:

- Proposing an oVirt counter

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/main/CONTRIBUTING.md): @sandrobonazzola 

This pull request needs review by: @oVirt/developers

### CODE REVIEW

File: source/community/ovirt-counter.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: oVirt Counter
+authors: sandrobonazzola
+page_classes: community
+---
+
+# oVirt Counter
+
+Once upon a time there was the [Linux Counter Project](https://en.wikipedia.org/wiki/Linux_Counter).
+If you want your oVirt deploymnet listed here you can edit this page follwoing the link on the bottom of this page.
```
Comment (mz-pdm) [inline source/community/ovirt-counter.md:10]: ... deployment ... following ...

Comment (sandrobonazzola) [reply source/community/ovirt-counter.md]: Thanks! fixed.

**Review by sanjacodes — APPROVED** [review]

File: source/community/ovirt-counter.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+authors: sandrobonazzola
+page_classes: community
+---
+
+# oVirt Counter
+
+Once upon a time there was the [Linux Counter Project](https://en.wikipedia.org/wiki/Linux_Counter).
+If you want your oVirt deployment listed here you can edit this page following the link on the bottom of this page.
+
+**IMPORTANT!** Please note oVirt project has no way to verify added data correspond to reality.
```
Comment (barpavel) [inline source/community/ovirt-counter.md:12]: Suggestions:
1. `oVirt project` --> `the oVirt project`
2. `added data` --> `that the added data` or `whether the added data`

Fix: `correspond` --> `corresponds`?

Comment (sandrobonazzola) [reply source/community/ovirt-counter.md:12]: Thanks, fixed.

File: source/community/ovirt-counter.md (hunk 1 above)
Comment (barpavel) [inline source/community/ovirt-counter.md:10]: `deploymnet`, `follwoing` regression :)

Comment (sandrobonazzola) [reply source/community/ovirt-counter.md:10]: :-/ will send a fix

Comment (sandrobonazzola) [reply source/community/ovirt-counter.md:10]: fixed in 2b18b8947e

**Pull request merged**